progenitor-impl/src/lib.rs

```diff
@@ -60,6 +61,43 @@ pub struct GenerationSettings {
     convert: Vec<(schemars::schema::SchemaObject, String, Vec<TypeImpl>)>,
 }
 
+impl Default for GenerationSettings {
+    fn default() -> Self {
+        // set Type from String to meta data for form body parts
+        use schemars::schema::{InstanceType, SchemaObject};
+        let bin_string = SchemaObject {
+            instance_type: Some(schemars::schema::SingleOrVec::Single(
+                Box::new(InstanceType::String),
+            )),
+            format: Some("binary".to_string()),
+            ..SchemaObject::default()
+        };
+        // This treats binary strings on the level of generated types.
+        // For now, we'd want to remove this field because
+        // the only supported binary string is handled as FormPart instead.
```

Comment:
"the only supported binary string" - maybe this needs rewording, as progenitor does support:
```yaml
      requestBody:
        content:
          application/octet-stream:
            schema:
              type: string
              format: binary
```
c.f. https://github.com/oxidecomputer/progenitor/blob/a6bc0624803b9483983f03269533cc35db66a54b/progenitor-impl/src/method.rs#L149